src/common/imagjpeg.cpp

```diff
@@ -35,6 +35,9 @@
 #endif
 
 #include "jpeglib.h"
+#if wxHAS_CXX17_INCLUDE("jversion.h")
```

Comment:
Do we really need this? It looks like this file exists since at least 1991, so why check for it?

Reply:
When I would just include the header file, I got errors on some of the UNIX builds saying that the file was missing. My guess was that these configurations weren't using the builtin libjpeg but the system one instead and for whatever reason this file was missing in the system header files. So, wrapping it in a `wxHAS_CXX17_INCLUDE` prevented that.

Reply:
Yes, sorry, you're right, it's not present in libjpeg installed from the package.

But the reason for this seems to be that it's an internal header which is not supposed to be used from the outside. Looking at `jerror.h`, it looks like there should be some way to get it as it's the message associated with (public) `JMSG_COPYRIGHT` code, can you please check if we can't get to it using public libjpeg API?